tests/network/vmi_multus.go

```diff
@@ -332,26 +332,41 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
+			getFedoraVmi := func(interfaces []v1.Interface, networks []v1.Network, cloudInitNetworkData string) *v1.VirtualMachineInstance {
+				withVmiOptions := []libvmi.Option{
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkData, false),
+				}
+				for idx := range networks {
+					withVmiOptions = append(withVmiOptions,
+						libvmi.WithInterface(interfaces[idx]),
+						libvmi.WithNetwork(&networks[idx]),
+						libvmi.WithCloudInitNoCloudUserData(tests.GetFedoraToolsGuestAgentUserData(), false),
+					)
+				}
+				return libvmi.NewTestToolingFedora(withVmiOptions...)
+			}
```

Comment:
I don't quite understand why you can't achieve this with an alpine VM. You'll have to provide a reason for changing the VM image.